### PR TITLE
Simplify Hitachi_Data_Systems.xml

### PR DESCRIPTION
**Pull request by cschanaj**

### CODE REVIEW

File: src/chrome/content/rules/Hitachi_Data_Systems.xml [hunk 1]
```diff
@@ -1,20 +1,16 @@
...
-	Nonfunctional subdomains:
-
-		- (www.)	(times out)
-		- channelone	(reset, expired 2013-02-08)
+	Non-functional hosts
+		Timeout was reached:
+		- channelone.hds.com
+		- portal.hds.com
 
+		Peer certificate cannot be authenticated with given CA certificates:
```
Comment (J0WI) [inline src/chrome/content/rules/Hitachi_Data_Systems.xml]: Cert was expired for me.

File: src/chrome/content/rules/Hitachi_Data_Systems.xml [hunk 2]
```diff
@@ -1,20 +1,16 @@
 <!--
-	Nonfunctional subdomains:
-
-		- (www.)	(times out)
-		- channelone	(reset, expired 2013-02-08)
+	Non-functional hosts
+		Timeout was reached:
+		- channelone.hds.com
+		- portal.hds.com
```
Comment (J0WI) [inline src/chrome/content/rules/Hitachi_Data_Systems.xml]: WFM

File: src/chrome/content/rules/Hitachi_Data_Systems.xml [hunk 3]
```diff
@@ -1,20 +1,16 @@
 <!--
-	Nonfunctional subdomains:
-
-		- (www.)	(times out)
-		- channelone	(reset, expired 2013-02-08)
+	Non-functional hosts
```
Comment (J0WI) [inline src/chrome/content/rules/Hitachi_Data_Systems.xml]: See also `qr.hds.com`

**Comment by cschanaj** [comment]
@J0WI fixed accordingly. thanks!!

**Pull request merged**